ReactWindows/ReactNative.Shared/UIManager/UIImplementation.cs

```diff
@@ -117,10 +117,11 @@ public class UIImplementation
         /// Unregisters a root view with the given tag.
         /// </summary>
         /// <param name="rootViewTag">The root view tag.</param>
-        public void RemoveRootView(int rootViewTag)
+        /// <param name="cleanupCompleted">Callback called when cleanup completes.</param>
+        public void RemoveRootView(int rootViewTag, Action cleanupCompleted)
```

Comment:
>RemoveRootView [](start = 20, length = 14)

Callbacks feel kind of like JavaScript. Any reason not to make this also async?

Reply:
I could, it's just that I get to this point where I'm enqueuing to operations queue, and I don't know how to keep it C#'ish

---
In reply to: [178548209](https://github.com/Microsoft/react-native-windows/pull/1737#discussion_r178548209) [](ancestors = 178548209)

Reply:
Hmm, I think it can be done, I'd just have to change the names a little. ok

---
In reply to: [178819919](https://github.com/Microsoft/react-native-windows/pull/1737#discussion_r178819919) [](ancestors = 178819919,178548209)